pkg/storage/stores/indexshipper/compactor/compactor.go

```diff
@@ -248,10 +262,10 @@ func (c *Compactor) init(objectClient client.ObjectClient, schemaConfig config.S
 			encoder = client.FSEncoder
 		}
 
-		chunkClient := client.NewClient(objectClient, encoder, schemaConfig)
+		c.chunkClient = client.NewClient(objectClient, encoder, schemaConfig)
```

Comment:
These are attached to the compactor so we can create `tableMarker` instances for individual compactions